quickstart/manifests/control-plane.yaml

```diff
@@ -551,7 +551,7 @@ spec:
               done;
       containers:
         - name: server
-          image: "ghcr.io/pipe-cd/pipecd:v0.52.2"
+          image: "ghcr.io/pipe-cd/control-plane:v0.52.2"
```

Comment:
Same reason as https://github.com/pipe-cd/pipecd/pull/6090/files#r2244401408

Also, I think until we cut the release, this control-plane image tag would not be available